specification/draft/index.html

```diff
@@ -905,89 +905,95 @@ <h3>Verifiable Presentation of DSCSA Stakeholder Credentials</h3>
+              the date from when on the Verifiable Presentation is valid.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>exp</b></td>
+            <td><b>`exp`</b></td>
             <td>
-              Expiration Date of the VC in EPOCH format
+              A digital wallet SHALL include an `exp` field in the JWT payload. This field represents the UNIX timestamp
+              expiration date of the Verifiable Presentation and SHALL be five minutes in the future based on the
+              issuance date.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>vp</b></td>
+            <td><b>`nonce`</b></td>
             <td>
-              Contains the Verifiable Presentation
+              A digital wallet SHALL include a `nonce` field in the JWT payload. This field represents a random value
+              that is used to prevent replay attacks.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#jwt-encoding</td>
           </tr>
           <tr>
-            <td><b>vp.type</b></td>
+            <td><b>`vp`</b>
+            </td>
+            <td>
+              A digital wallet SHALL include a `vp` object in the JWT payload. This object contains further data of
+              the Verifiable Presentation.
+            </td>
+          </tr>
+          <tr>
+            <td><b>`vp.@context`</b></td>
             <td>
-              The type property is required and expresses the type of presentation, such as VerifiablePresentation.
+              A digital wallet SHALL include a `@context` field in the `vp` object. This field represents the context
+              of the Verifiable Presentation and SHALL be set to `https://www.w3.org/2018/credentials/v1`.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#presentations-0</td>
           </tr>
           <tr>
-            <td><b>vp.verifiableCredential</b></td>
+            <td><b>`vp.type`</b></td>
             <td>
-              The VerifiableCredential (including all attributes) that are represented by this VP
+              A digital wallet SHALL include a `type` field in the `vp` object. This field represents the type of
+              the Verifiable Presentation and SHALL be set to `VerifiablePresentation`.
             </td>
-            <td>https://w3c.github.io/vc-data-model/#presentations-0</td>
           </tr>
           <tr>
-            <td><b>nonce</b></td>
+            <td><b>`vp.verifiableCredential`</b></td>
             <td>
-              Contains the corrUUID for which the VP has been generated for.
+                A digital wallet SHALL include a `verifiableCredential` field in the `vp` object. This field contains
+                the embedded ATP Verifiable Credential.
             </td>
```

Comment:
so, did we lose a field that contains the corrUUID? not needed anymore?

re spelling: 

> `verifiableCredential` field 

>> pls check capitalisations. seems to be a mix here.